sport/app/football/feed/Competitions.scala

```diff
@@ -101,6 +100,14 @@ trait Competitions extends implicits.Football {
 
   def matches: Seq[FootballMatch] = competitions.flatMap(_.matches)
 
+  def isAMatchInProgress(matches: Seq[FootballMatch], clock: Clock): Boolean =
+    matches.exists(game => {
+      val gameUtc = game.date.withZoneSameInstant(ZoneOffset.UTC) // game.date.withZoneSameInstant(ZoneOffset.UTC)
+      val currentTimeUtc = clock.instant.atZone(ZoneOffset.UTC)
```

Comment:
Here I'm using `clock.instant.atZone(ZoneOffset.UTC)` to get the current utc time. 

I could have done this by `ZonedDateTime.now(clock.getZone).withZoneSameInstant(ZoneOffset.UTC)`

But that way I would have had problem in testing because now I am controlling the current time in the test

Reply:
This was changed as recommended by Adam. and now we're doing `ZonedDateTime.now(clock)` to be consistent with the rest of the code